core-common/src/main/java/org/zalando/nakadi/domain/EventAuthField.java

```diff
@@ -0,0 +1,60 @@
+package org.zalando.nakadi.domain;
+
+import javax.validation.constraints.NotNull;
+
+public class EventAuthField {
+    @NotNull
+    private String type;
+
+    @NotNull
+    private String path;
```

Comment:
Maybe it makes sense to set validation min. length to 1, as empty field doesn't make sense (but I am not sure)